packages/x-components/src/x-modules/experience-controls/store/module.ts

```diff
@@ -1,6 +1,20 @@
+import { ExperienceControlsXStoreModule } from './types';
+
 /**
  * {@link XStoreModule} For the empathize module.
  *
  * @internal
  */
-export const experienceControlsXStoreModule: any = {};
+export const experienceControlsXStoreModule: ExperienceControlsXStoreModule = {
+  state: () => ({
+    controls: {},
+    events: {}
+  }),
+  getters: {},
+  mutations: {
+    setControls(state: any, experienceControlsConfig: any) {
```

Comment:
Don't force types here, they should be infered from `ExperienceControlsXStoreModule`.

```suggestion
    setControls(state, controls) {
```